Documentation/sifive/setup.md

```diff
@@ -69,14 +69,11 @@ sudo apt-get install autoconf automake autotools-dev curl libmpc-dev \
 git clone https://github.com/riscv/riscv-gnu-toolchain --recursive
```

Comment:
Unrelated to this patch, why not use the upstream toolchain instead of this fork?